site/docs/use-cases/1-LLM-pipeline/_sections/_convert_model.mdx

```diff
@@ -0,0 +1,71 @@
+## Convert and Optimize Model
+
+<Tabs groupId="model-source">
+    <TabItem label="From Hugging Face" value="huggingface">
+        Use `optimum-intel` package to convert and optimize models:
+        ```bash
+        pip install optimum-intel[openvino]
+        ```
+
+        Download and convert a model to the OpenVINO IR format:
+        <Tabs groupId="export-precision">
+            <TabItem label="Compress weights to the int4 precision" value="int4">
+                ```bash
+                optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf --trust-remote-code --weight-format int4 ov_llama_2_7b_int4
+                ```
+            </TabItem>
+            <TabItem label="Keep full model precision" value="fp16">
+                ```bash
+                optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf --trust-remote-code --weight-format fp16 ov_llama_2_7b_fp16
```

Comment:
The difference of the cmd is in the end. The end isn't visible on my screen so the tabs look identical until I scroll them. Move the alternating flags to the beginning

Reply:
Fixed